### PR TITLE
feat: very slight gradient background via withBackground hoc

### PR DESCRIPTION
**Pull request by logan-r**
### Tickets:

- HCK- #876 

### List of changes:

- Very slight (#fff to #f4f4f4) gradient background for pages, addable by wrapping a page in `withBackground()`

### Type of change:

Please delete options that aren't relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots

### CODE REVIEW

**Review by Tony9984 — COMMENTED** [review]
looks good! there's a small issue tho the container seems to be overflowing on the login page, i'm able to scroll down as shown here:
image

**Comment by logan-r** [comment]
Yeah, the login page needed it's DOM tree structure redone, currently finished in https://github.com/hackmcgill/dashboard/pull/872:
image

File: src/pages/Login/index.tsx [hunk 1]
```diff
@@ -156,16 +156,16 @@ const LoginPage: React.FC = () => {
...
-          </div>
-        )
+            <>
+              {renderForm()}
+              <BackgroundImage
+                src={Coders}
+                top={'60px'}
+                right={'0px'}
+                imgHeight={'100%'}
+              />
```
Comment (Tony9984) [inline src/pages/Login/index.tsx:166]: i managed to fix the scroll using a height of 'calc(100% - 60px)', also the image seems to be below the background

**Review by Tony9984 — COMMENTED** [review]
> #872

ohhhhhh alright got it should be good then!

File: src/pages/Login/index.tsx (hunk 1 above)
Comment (logan-r) [reply src/pages/Login/index.tsx:166]: Fix in #872

**Review by Tony9984 — APPROVED** [review]

**Pull request merged**